### ISSUE TITLE
When the OCR-PROCESSING fails users can not see the scan as it is still a pdf

### ISSUE THREAD

**Issue by anlausch** [locdb/locdb-frend#312]
Maybe we need a more flexible scan viewer

**Comment by lgalke** [comment]
I agree that when processing hasnt finished yet we could show the pdf in-between, but to implement it, we would need a flag that tells us whether the processing is finished or not. Is there any indicator that tells us whether the png file is ready?  For now, we only retrieve something at `/scans/<scan_id>`, of which we do not know whether it is still a pdf or a png already.

**Comment by anlausch** [comment]
You can use the filename extension in order to check for the filename.

The case also applies to the view when it is still ocr processing and the user wants to check the file.

**Comment by lgalke** [comment]
At the moment, there is no extension in the URL that is assembled in the front end: `${this.locdbUrl}/scans/${identifier}`.

So there are actually `${identifier}.png` and `${identifier}.pdf` files?

Please specify, then we'll find a way how to handle those cases.

**Comment by lgalke** [comment]
We can use the `name` property of the scans to determine whether its a pdf (OCR not finished) or a png (OCR finished). We should display at least an error (or disable the scan in the dropdown, if its pdf).

**Comment by lgalke** [comment]
It should work right now, just needs some testing.
